packages/engine/src/output/buffer/mod.rs

```diff
@@ -22,7 +18,7 @@ use crate::{
     },
 };
 
-const RELATIVE_PARTS_FOLDER: &str = "./parts";
+pub(crate) const RELATIVE_PARTS_FOLDER: &str = "./parts";
```

Comment:
I don't like the idea of making this more public. It's only used in this module and should be exposed.

Reply:
"should be exposed", I don't understand

Reply:
Sorry, meant to be "shouldn't be exposed".

Reply:
Ok yeah, makes sense.
Something like https://github.com/hashintel/hash/pull/494/commits/8174c718cc2ea17292d3202e45d601af644f334e